db/schema.rb

```diff
@@ -1,3 +1,26 @@
-ActiveRecord::Schema.define(version: 0) do
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_02_04_115643) do
+
+  # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+
+  create_table "trips", force: :cascade do |t|
+    t.string "name"
+    t.string "coord_lat"
```

Comment:
Why not call the attribute `coords` and store the lat and long as a hash? (see Cooper Challenge)

Reply:
This is because of the problems that hashes can cause with put and patch actions. Was a pro-active measure.

Reply:
In the most recent project the herald we had an issue with nested objects as params. 
We solved it with a workaround you showed us      

```params: {
        "[article][published]": false
      }```

but that was a workaround. 

more info on the issue: https://github.com/axios/axios/issues/738

to avoid this the idea is to split the coords